iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java

```diff
@@ -261,6 +268,13 @@ public boolean nextKeyValue() throws IOException {
       while (true) {
         if (currentIterator.hasNext()) {
           current = currentIterator.next();
+          Configuration conf = context.getConfiguration();
```

Comment:
Is the context changing here? Shall we move the conf object to outside of the loop?